src/styles/utils/variables.scss

```
@@ -0,0 +1,4 @@
+$main-color: #060b35;
+$second-color: #616070;
+$third-color: #00acdc;
```

not as pretty names as they could be
maybe you don't mean "second-color" but "secondary-color".
and the name "third-color" do not say anything meaningful

for example, such names as `accent`, `shadow`, `warning`... - say something valuable, but `third' does not

find right names is an important part of  good programming skills
```suggestion
$secondary-color: #616070;
$third-color: #00acdc; please give some valuable name
```